action.yml

```diff
@@ -156,20 +181,30 @@ runs:
         GRADLE_PUBLISH_KEY: ${{ inputs.gradle-publish-key }}
         MAVEN_CENTRAL_PASSWORD: ${{ inputs.maven-central-password }}
         MAVEN_CENTRAL_USERNAME: ${{ inputs.maven-central-username }}
+        SIGNING_PASSWORD: ${{ inputs.signing-password }}
+        #
+        ORG_GRADLE_PROJECT_mavenRepo: ${{ inputs.maven-central-repo }}
+        ORG_GRADLE_PROJECT_mavenUsername: ${{ inputs.maven-central-password }}
+        ORG_GRADLE_PROJECT_mavenPassword: ${{ inputs.maven-central-username }}  
         ORG_GRADLE_PROJECT_signingKey: ${{ inputs.signing-key }}
         ORG_GRADLE_PROJECT_signingPassword: ${{ inputs.signing-password }}
-        SIGNING_PASSWORD: ${{ inputs.signing-password }}
+        ORG_GRADLE_PROJECT_npmRepo: ${{ inputs.npm-repo }}
+        ORG_GRADLE_PROJECT_npmToken: ${{ inputs.npm-token }}
```

Comment:
These are nice. I would also expose them as `NPM_REPO` and `NPM_TOKEN`, consistently with how we deal with all other variables